ert3/algorithms/_sensitivity.py

```diff
@@ -0,0 +1,28 @@
+def _build_base_records(groups, parameters):
+    return {group_name: parameters[group_name].ppf(0.5) for group_name in groups}
+
+
+def one_at_the_time(parameters):
+    tail = (1 - 0.99) / 2
```

Comment:
I understand where this number comes from now, but I think it warrants a comment for future readability 🙂

Reply:
Think it also should be moved below the "assertion" on the parameter length.

Reply:
Added a comment and moved the variable definition below the assert ✅